pkg/operator/controller/names.go

```diff
@@ -36,6 +36,12 @@ const (
 	// DefaultCanaryNamespace is the default namespace for
 	// the ingress canary check resources.
 	DefaultCanaryNamespace = "openshift-ingress-canary"
+
+	// CanaryHealthcheckCommand is a parameter to pass to the ingress-operator to call
```

Comment:
I think it might make more sense to move these constants to the canary controller, since these aren't actually names. What do you think? We can import the canary package wherever these names may be needed.

Reply:
Also should `CanaryHealthcheckCommand` be `CanaryHealthCheckCommand`?

Reply:
I had them there originally, but changed my mind because I thought they made more sense with the consts in names. I will move them back, not a big deal.

Reply:
Healthcheck is expressed as one word here, instead of two words.

Reply:
sounds good. Thanks!